src/test/java/teammates/test/cases/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -795,4 +797,39 @@ private InstructorCourseEditPage getCourseEditPage() {
         return loginAdminToPage(courseEditPageLink, InstructorCourseEditPage.class);
     }
 
+    private void testEditInstructorDisplayedToStudents() {
```

Comment:
We can have multiple _test separator_ lines, like this:

```java
    private void testEditInstructorDisplayedToStudents() {

        instructorId = testData.instructors.get("InsCrsEdit.instructor1OfCS2105").googleId;
        courseId = testData.courses.get("InsCrsEdit.CS2105").getId();
        courseEditPage = getCourseEditPage();

        ______TS("verify sole visible instructor cannot be made invisible");
        courseEditPage.editInstructor(1, "Teammates Do Not Save", "insCrsEdit.instructor1@cs2105.tmt",
                false, "CS2105 Instructor 1", "Co-owner");

       courseEditPage.verifyInstructorDetails(1, "CS2105 Instructor 1",
                "insCrsEdit.instructor1@cs2105.tmt", true, "CS2105 Instructor 1", "Co-owner");
        courseEditPage.verifyStatus(String.format(Const.StatusMessages.COURSE_INSTRUCTOR_NO_INSTRUCTOR_DISPLAYED));

        ______TS("verify an invisible instructor can be made visible");
        courseEditPage.editInstructor(2, "CS2105 Instructor 2", "new_email_number_2@email.tmt",
                false, "CS2105 Instructor 2", "Co-owner");
        courseEditPage.verifyInstructorDetails(2, "CS2105 Instructor 2", "new_email_number_2@email.tmt",
                false, "CS2105 Instructor 2", "Co-owner");
        courseEditPage.verifyStatus(String.format(Const.StatusMessages.COURSE_INSTRUCTOR_EDITED, "CS2105 Instructor 2"));

         ______TS("verify an instructor can be made invisible if there are other visible instructors");
        courseEditPage.editInstructor(2, "CS2105 Instructor 2", "insCrsEdit.instructor2@cs2105.tmt",
                true, "CS2105 Instructor 2", "Co-owner");
        // set instructor 1 to invisible and also change email (for variety)
        courseEditPage.editInstructor(1, "CS2105 Instructor 1", "new_email@email.tmt", false,
                "CS2105 Instructor 1", "Co-owner");
        courseEditPage.verifyInstructorDetails(1, "CS2105 Instructor 1", "new_email@email.tmt",
                false, "CS2105 Instructor 1", "Co-owner");
        courseEditPage.verifyStatus(String.format(Const.StatusMessages.COURSE_INSTRUCTOR_EDITED, "CS2105 Instructor 1"));
    }
```
Note how some comments have been removed/edited.

Reply:
updated